panopticum/management/commands/config/default.json

```diff
@@ -0,0 +1,305 @@
+{

```

Comment:
Why we need this file? Why can't we live with data only in csv file?

Reply:
I began with CSV file which cannot have multiple sheets to represent seed data for all tables effectively. Hence came up with defaults.json.

It will converge to one xlsx file in the next version with different component details format. Till then we'll have 2 files with input data.